spanner/metrics.go

```diff
@@ -228,18 +277,31 @@ func newBuiltinMetricsTracerFactory(ctx context.Context, dbpath string, metricsP
 		},
 		shutdown: func(ctx context.Context) {},
 	}
-
+	tracerFactory.isAFEMetricsEnabled = strings.EqualFold("false", os.Getenv("SPANNER_DISABLE_AFE_SERVER_TIMING"))
```

Comment:
Is the reason that we have named the env var `SPANNER_DISABLE_AFE_SERVER_TIMING` (instead of `ENABLE`) that it will be enabled by default later? The current requirement to enable it by setting `DISABLE=false` sounds weird, but I understand it if it later will be enabled by default.

Reply:
Yes. These new metrics will be enabled by default to make sure stats are always available when we need them.